cppcache/src/TcrConnection.cpp

```diff
@@ -47,10 +46,10 @@ const int8_t LAST_CHUNK_MASK = 0x1;
 const int64_t INITIAL_CONNECTION_ID = 26739;
 
 #define throwException(ex)                            \
-  {                                                   \
+  do {                                                \
```

Comment:
haven't looked at in c++ in a while (no pun intended) - what is the rationale for the do/while here?

Reply:
Fixes the multi line macro issue with semicolons. It gets compiled out.